Sources/RouterService/RouterService.swift

```diff
@@ -56,8 +56,22 @@ public final class RouterService: RouterServiceProtocol, RouterServiceRegistrati
             fromViewController: viewController
         )
         let destinationFeature = destinationFeatureType.initialize(withStore: store)
+        let destinationViewController: UIViewController
+        
+        if destinationFeature.isEnabled() {
+            destinationViewController = destinationFeature.build(fromRoute: route)
+        } else {
+            let fallbackFeatureType = destinationFeature.fallback(forRoute: route)
+            guard let fallbackDestinationFeature = fallbackFeatureType?.initialize(withStore: store) else {
+                failureHandler()
```

Comment:
```suggestion
                print("Attempted to initialize a disabled feature, but it has no fallback!")
                failureHandler()
```

It was my mistake to make this failureHandler not take a string... We can just print now and in the future improve it to take a message as a parameter (so real crashes will print the message)

Reply:
We decide to do this improvement this failure handler in another (soon as possible) Pull Request!